app/models/user-registration.ts

```diff
@@ -83,6 +77,14 @@ export default class UserRegistrationModel extends Model.extend(Validations) {
     existingEmails: Set<string> = new Set();
     invalidEmails: Set<string> = new Set();
 
+    get existingEmailsArray(): string[] {
+        return [...this.existingEmails];
+    }
+
+    get invalidEmailsArray(): string[] {
+        return [...this.invalidEmails];
+    }
+
```

Comment:
If you use `alias`, do you still need these getters?

Reply:
NVM. I was thinking about just using the alias on `existingEmails` and `invalidEmails`, but they are `Set`s so might not be straigtforward to do.